browser/pages/location/location.html

```diff
@@ -23,21 +23,29 @@
                  ng-init="locCtrl.checkFolder()" ng-change="locCtrl.checkFolder()" ng-model="locCtrl.folder" path-validator autofocus/>
           <button id="location-browse-btn" type="button" class="btn btn-primary location-browse-btn" ng-click="locCtrl.selectFolder()">Browse</button>
 
-          <div ng-show="!locationForm.folder.$invalid && locCtrl.folderExists" class="has-warning">
+          <div ng-show="!locationForm.folder.$invalid && locCtrl.folderExists && !locCtrl.nonasciichars" class="has-warning">
             <strong id="existingFolderStatus" class="help-block">
               <span class="pficon pficon-warning-triangle-o"></span>
               The selected folder already exists, some files might be overwritten.
             </strong>
           </div>
 
-          <div ng-show="!locationForm.folder.$invalid && !locCtrl.folderExists">
+          <div ng-show="!locationForm.folder.$invalid && !locCtrl.folderExists && !locCtrl.nonasciichars">
             <strong id="createFolderStatus" class="help-block">
               <span class="pficon pficon-info"></span>
               The selected folder will be created.
             </strong>
           </div>
 
-          <div ng-show="locationForm.folder.$error" class="help-block has-error">
+          <div ng-show="locCtrl.checkUserProfilestatus() && locCtrl.nonasciichars" class="help-block has-error">
+            <strong id="checkUserProfilestatus" class="help-block has-error">
+              <span class="pficon-error-circle-o"></span>
+              Path to user home directory has Non-ASCII characters
```

Comment:
You can extract MSI in different location, but when you run msi as 

``` shell 
msiexec /i installer.msi
```

there is no good way to control where msiexec extracts .cab files during installation. They go to %TMP% and code below does not work, because MSI seems to read %TMP% directly form configuration not from current session. For example code below does not work.

``` shell
set TMP=c:\Temp
msiexec /i installer.msi
```

msi installer itself does not have public property to override TMP folder location.

We can only set TMP env var directly in settings using powershell, but that would require to restore original value after installation is over. It is also not easy to do, because original value normally %USERPROFILE%/AppData/Local/Temp and to save that string without %VAR% expansion it should be read directly form registry, so it is a little to much :)

I am out of ideas and open for any working solution.